dockers/docker-database/flush_db.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Wait until redis starts
+until [[ $(sonic-db-cli PING | grep -c PONG) -gt 0 ]]; do
+  sleep 1;
+done
+
+DUMPFILE=/var/lib/dump.rdb
+if [[ -f $DUMPFILE ]]; then
+    flush_warmboot_database
```

Comment:
>flush_warmboot_database [](start = 4, length = 23)

why merge it into this script as a function?

Reply:
flush_warmboot_database is a python script codes, we don't want to mix it with shell cmds. If we rewrite those shell cmds into python codes, it is a redundancy， for example, sonic-db-cli is already a python script.

Reply:
Better to rewrite this in python, so merge should be easier.

---
In reply to: [520939574](https://github.com/Azure/sonic-buildimage/pull/5773#discussion_r520939574) [](ancestors = 520939574)

Reply:
merge into on python script